.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,7 +5,7 @@ This PR fixes #
 **Notes for Reviewers**
 
 
-**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
+**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
```

Comment:
Let not change this line . If there is any endpoint with **layer5io** then only change it to **meshery** rest keep as it is.